{
	"id": "oval:org.debian:def:310809397459454144451376538357853871038",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2021-33571 python-django",
		"affected": {
			"family": "unix",
			"platform": "Debian GNU/Linux 12",
			"product": "python-django"
		},
		"reference": [
			{
				"source": "CVE",
				"ref_id": "CVE-2021-33571",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-33571"
			}
		],
		"description": "In Django 2.2 before 2.2.24, 3.x before 3.1.12, and 3.2 before 3.2.4, URLValidator, validate_ipv4_address, and validate_ipv46_address do not prohibit leading zero characters in octal literals. This may allow a bypass of access control that is based on IP addresses. (validate_ipv4_address and validate_ipv46_address are unaffected with Python 3.9.5+..) .",
		"debian": {}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.debian.oval:tst:2",
								"comment": "all architecture"
							},
							{
								"test_ref": "oval:org.debian.oval:tst:21051",
								"comment": "python-django DPKG is earlier than 2:2.2.24-1"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.debian.oval:tst:1",
				"comment": "Debian 12 is installed"
			}
		]
	}
}
